src/app/orbit-list/orbit-list.component.html

```diff
@@ -6,15 +6,28 @@
         <th>Orbit Type</th>
         <th>Launch Date</th>
     </tr>
-    <tr class="data-row">
+    <!-- <tr class="data-row">
         <td>GPS 938</td>
         <td>Positioning</td>
         <td>true</td>
         <td>HIGH</td>
         <td>2001-11-01</td>
+    </tr> -->
+
+    <!-- Displaying satellite data in the table using *ngFor -->
+    <tr *ngFor=" let satellite of satellites">
+        <td>{{satellite.name}}</td>
+        <!-- Adding style directive to conditionally highlight satellite type = 'Space Debris' -->
+        <td [class.warning]="satellite.isSpaceDebris()">{{satellite.type}}</td>
+        <td>{{satellite.launchDate}}</td>
+        <td>{{satellite.orbitType}}</td>
+        <td>{{satellite.operational}}</td>
```

Comment:
```suggestion
        <td>{{satellite.operational}}</td>
        <td>{{satellite.orbitType}}</td>
        <td>{{satellite.launchDate}}</td>
```
When loading your page, I noticed that your `launchDates` were being rendered under the Operational column and your `operational` values rendered under the Launch Date column. I do not think this is causing anything to break, but I would recommend switching them to ensure it behaves as expected.